jaq-interpret/src/error.rs

```diff
@@ -55,7 +55,7 @@ impl Error {
     }
 
     /// Build an error from something that can be converted to a string.
-    pub fn str(s: impl ToString) -> Self {
+    pub fn str(s: &impl ToString) -> Self {
```

Comment:
I think that we cannot do this change, because this is part of the public API.

Reply:
Reverted in https://github.com/01mf02/jaq/pull/124/commits/4828b4f84b4cf0b530909c3e51d1a3afdd999657